src/css/objects/_media.scss

```diff
@@ -245,172 +241,216 @@
 // We're bending some BEM rules here by having 1-deep nested selectors, but in
 // exchange for that we get a single common markup pattern for media objects.
 
-.media--last {
-  margin-bottom: 0;
-}
-
-.media--new-and-unread {
-
-  .media__headline {
-    position: relative;
-
-    &::before {
-      background-color: $color-primary;
-      border-radius: 50%;
-      content: '';
-      display: block;
-      height: 0.5rem;
-      left: ($base-line-height / -2);
-      position: absolute;
-      top: 1rem;
-      width: 0.5rem;
-    }
+.o-media--sm {
+  align-items: center;
+  display: flex;
+  margin: 0 0 $base-line-height;
 
+  .o-media__headline, .o-media__title {
+    margin: 0;
   }
 
-  .media__indicator-visited {
-    display: none;
+  > .o-media__figure {
+    margin-right: 1.125rem;
+    max-width: 5rem;
+    float: left;
+    margin-right: 3.55505%;
+    width: 4.5rem;
```

Comment:
What's the rationale for using `rem` instead of percentages to express the width of floated figures?